modules/Schedulers/language/en_us.lang.php

```diff
@@ -139,16 +139,12 @@
     'LNK_NEW_SCHEDULER' => 'Create Scheduler',
     'LNK_LIST_SCHEDULED' => 'Scheduled Jobs',
 // Messages
-    'ERR_DELETE_RECORD' => 'You must specify a record number to delete the schedule.',
     'ERR_CRON_SYNTAX' => 'Invalid Cron syntax',
-    'NTC_DELETE_CONFIRMATION' => 'Are you sure you want to delete this record?',
-    'NTC_STATUS' => 'Set status to Inactive to remove this schedule from the Scheduler dropdown lists',
     'NTC_LIST_ORDER' => 'Set the order this schedule will appear in the Scheduler dropdown lists',
     'LBL_CRON_INSTRUCTIONS_WINDOWS' => 'To Setup Windows Scheduler',
     'LBL_CRON_INSTRUCTIONS_LINUX' => 'To Setup Crontab',
     'LBL_CRON_LINUX_DESC1' => 'In order to run SuiteCRM Schedulers, edit your web server user\'s crontab file with this command: ',
     'LBL_CRON_LINUX_DESC2' => '... and add the following line to the crontab file: ',
-    'LBL_CRON_LINUX_DESC3' => 'You should do this only after the installation is concluded.',
```

Comment:
@Dillon-Brown the string `'LBL_CRON_LINUX_DESC3' ` is still in use here:
https://github.com/salesagility/SuiteCRM/blob/hotfix/install/ready.php#L292